client/src/components/requestPanel/RequestSender.tsx

```diff
@@ -181,18 +181,22 @@ function RequestSender({
 
     const collection = collections.find((c) => c.id === request.collectionId);
 
-    const collectionHeaders = collection?.data?.headers ?? [];
+    const enabledCollectionHeaders = collection?.data?.headers
+      ? collection.data.headers.filter((h) => {
+          return h.isEnabled !== false;
```

Comment:
you don't need

```
(h) => {
          return h.isEnabled !== false;
}
```

you can just write



```
(h) => !h.isEnabled
```

Reply:
I think here it's not the case. 
We want only to filter out cases where `h.isEnabled !== false`, meaning we want to use headers, where isEnabled is not explicitly set to false. In other words, we want to use headers where `isEnabled === true` or `isEnabled === undefined`.

Using `!h.isEnabled` on the other hand, would use headers with `isEnabled === false` and `isEnabled === undefined`.

Reply:
true, but we don't need the parentheses and return. Fixed:

```javascript
(h) => h.isEnabled !== false
```